<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton7">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton8">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton9">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_dev">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QComboBox" name="memoryList">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
       <property name="maxCount">
        <number>50</number>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="delButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_revert">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;--</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_erase">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton4">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton5">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton6">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_mul">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_lfbracket">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_rbracket">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="moduleButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>|x|</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="cosButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="sinButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="tanButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton1">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton2">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton3">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_min">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_step">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="sqrtButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="factButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>x!</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="arccosButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>arccos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="arcsinButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>arcsin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QPushButton" name="arctanButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>arctan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton0">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_per">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="3">
      <widget class="QPushButton" name="pushButton_eq">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>=</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="eButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="logButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="lnButton">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
